django/core/jinja2/common.jinja

```diff
@@ -301,16 +301,26 @@ Currently in <em><mark>{{ constants.DEPLOY_ENVIRONMENT }}</mark></em> mode.
 {{ date.strftime('%B %d, %Y %I:%M %p') }}
 {% endmacro %}
 
-{% macro render_ogp_tags(url=None, title=None, description=None, image=None) %}
-    <!-- Open Graph Protocol Tags -->
-
+{%
+    macro render_ogp_tags(
+        request,
+        title="The Network for Computational Modeling in the Social and Ecological Sciences (CoMSES Net)",
+        description="CoMSES Net is an international open research community dedicated to fostering good practices for computational / agent based modeling.",
+        image=None
+    )
+%}
+    {% set url = request.build_absolute_uri() %}
+    {% set absolute_image_url = request.build_absolute_uri(image or static("images/logo-comses.png")) %}
```

Comment:
should come up with a better default image but that can be done later as well: 

https://mastodon.social/@comses